docs/mapping-arrays.md

```diff
@@ -0,0 +1,27 @@
+# Mapping Arrays
+
+With the lack of constraints on `TSource` and `TDestination`, you can always provide a profile that expects or returns an array.
+
+```ts
+import { Profile } from '@kitbag/mapper'
+
+export const arrayToSet = {
+  sourceKey: 'array',
+  destinationKey: 'Set',
+  map: function <T>(source: T[]): Set<T> {
+    return new Set<T>(source)
+  },
+} as const satisfies Profile
+```
+
+However, if your goal is use the same mapping profile over an array of sources you can use either
+
+```ts
+const mapped = [123, 456].map((source) => mapper.map('number', source, 'Date'))
+```
+
+or for an even simpler solution, Kitbag Mapper provides `mapMany`, which takes an array of `TSource` and returns an array of `TDestination`.
+
+```ts
+const mapped = mapper.mapMany('number', [123, 456], 'Date')
+```
```

Comment:
I kinda like how you're showing that its super simple js to just use array.map but I might omit the first example and just highlight how easy `mapper.mapMany` is. 

Side note. Just crossed my mind that `mapAll` might be a nice name for that as well.

Reply:
agreed with removing example ✅ 

don't feel strongly about renaming. Part of me thinks we should just remove this method altogether, since the single `map` is still pretty straight forward

```ts
const mapped = [123, 456].map((source) => mapper.map('number', source, 'Date'))
```

I also kinda like

```ts
const mapToDate = (source: number) => mapper.map('number', source, 'Date')
const mapped = [123, 456].map(mapToDate)
```

Reply:
I think its a common enough pattern and its pretty simple to have `mapper.mapMany` so I think we should keep it. Its a little bit nicer syntax with less boilerplate. 

Lets just leave it as is 👍